### ISSUE TITLE
naqanet_dua_2019 pretrained model crashes on master

### ISSUE THREAD

**Issue by Phirefly9** [allenai/allennlp#2772]
**Describe the bug**
Attempting to use the naqanet_dua_2019 model results in a crash on the master branch

**To Reproduce**
run the following code
```
from allennlp.pretrained import naqanet_dua_2019
predictor = naqanet_dua_2019()
```

**Expected behavior**
It should not crash

**System (please complete the following information):**
 - OS: [Linux] ubuntu 16.04
 - Python version: [ 3.7.3 ]
 - AllenNLP version: [ "I installed from master"]
 - PyTorch version: (came with pip install of allennlp)

**Additional context**
the error message is
```
RuntimeError: Error(s) in loading state_dict for NumericallyAugmentedQaNet:
	Missing key(s) in state_dict:
```

followed by what appears to be pretty much the entire network description

**Comment by nelson-liu** [comment]
It'd be helpful if you could include the whole trace, particularly the bit right after `Missing key(s) in state_dict:` so we can see which keys are lacking.

**Comment by Phirefly9** [comment]
```
untimeError: Error(s) in loading state_dict for NumericallyAugmentedQaNet:
	Missing key(s) in state_dict: "_phrase_layer._encoder_blocks.0._conv_norm_layers.0.weight", "_phrase_layer._encoder_blocks.0._conv_norm_layers.0.bias", "_phrase_layer._encoder_blocks.0._conv_norm_layers.1.weight", "_phrase_layer._encoder_blocks.0._conv_norm_layers.1.bias", "_phrase_layer._encoder_blocks.0._conv_norm_layers.2.weight", "_phrase_layer._encoder_blocks.0._conv_norm_layers.2.bias", "_phrase_layer._encoder_blocks.0._conv_norm_layers.3.weight", "_phrase_layer._encoder_blocks.0._conv_norm_layers.3.bias", "_phrase_layer._encoder_blocks.0._conv_layers.0.1.weight", "_phrase_layer._encoder_blocks.0._conv_layers.0.1.bias", "_phrase_layer._encoder_blocks.0._conv_layers.0.2.weight", "_phrase_layer._encoder_blocks.0._conv_layers.0.2.bias", "_phrase_layer._encoder_blocks.0._conv_layers.1.1.weight", "_phrase_layer._encoder_blocks.0._conv_layers.1.1.bias", "_phrase_layer._encoder_blocks.0._conv_layers.1.2.weight", "_phrase_layer._encoder_blocks.0._conv_layers.1.2.bias", "_phrase_layer._encoder_blocks.0._conv_layers.2.1.weight", "_phrase_layer._encoder_blocks.0._conv_layers.2.1.bias", "_phrase_layer._encoder_blocks.0._conv_layers.2.2.weight", "_phrase_layer._encoder_blocks.0._conv_layers.2.2.bias", "_phrase_layer._encoder_blocks.0._conv_layers.3.1.weight", "_phrase_layer._encoder_blocks.0._conv_layers.3.1.bias", "_phrase_layer._encoder_blocks.0._conv_layers.3.2.weight", "_phrase_layer._encoder_blocks.0._conv_layers.3.2.bias", "_phrase_layer._encoder_blocks.0.attention_norm_layer.weight", "_phrase_layer._encoder_blocks.0.attention_norm_layer.bias", "_phrase_layer._encoder_blocks.0.attention_layer._combined_projection.weight", "_phrase_layer._encoder_blocks.0.attention_layer._combined_projection.bias", "_phrase_layer._encoder_blocks.0.attention_layer._output_projection.weight", "_phrase_layer._encoder_blocks.0.attention_layer._output_projection.bias", "_phrase_layer._encoder_blocks.0.feedforward_norm_layer.weight", "_phrase_layer._encoder_blocks.0.feedforward_norm_layer.bias", "_phrase_layer._encoder_blocks.0.feedforward._linear_layers.0.weight", "_phrase_layer._encoder_blocks.0.feedforward._linear_layers.0.bias", "_phrase_layer._encoder_blocks.0.feedforward._linear_layers.1.weight", "_phrase_layer._encoder_blocks.0.feedforward._linear_layers.1.bias", "_modeling_layer._encoder_blocks.0._conv_norm_layers.0.weight", "_modeling_layer._encoder_blocks.0._conv_norm_layers.0.bias", "_modeling_layer._encoder_blocks.0._conv_norm_layers.1.weight", "_modeling_layer._encoder_blocks.0._conv_norm_layers.1.bias", "_modeling_layer._encoder_blocks.0._conv_layers.0.1.weight", "_modeling_layer._encoder_blocks.0._conv_layers.0.1.bias", "_modeling_layer._encoder_blocks.0._conv_layers.0.2.weight", "_modeling_layer._encoder_blocks.0._conv_layers.0.2.bias", "_modeling_layer._encoder_blocks.0._conv_layers.1.1.weight", "_modeling_layer._encoder_blocks.0._conv_layers.1.1.bias", "_modeling_layer._encoder_blocks.0._conv_layers.1.2.weight", "_modeling_layer._encoder_blocks.0._conv_layers.1.2.bias", "_modeling_layer._encoder_blocks.0.attention_norm_layer.weight", "_modeling_layer._encoder_blocks.0.attention_norm_layer.bias", "_modeling_layer._encoder_blocks.0.attention_layer._combined_projection.weight", "_modeling_layer._encoder_blocks.0.attention_layer._combined_projection.bias", "_modeling_layer._encoder_blocks.0.attention_layer._output_projection.weight", "_modeling_layer._encoder_blocks.0.attention_layer._output_projection.bias", "_modeling_layer._encoder_blocks.0.feedforward_norm_layer.weight", "_modeling_layer._encoder_blocks.0.feedforward_norm_layer.bias", "_modeling_layer._encoder_blocks.0.feedforward._linear_layers.0.weight", "_modeling_layer._encoder_blocks.0.feedforward._linear_layers.0.bias", "_modeling_layer._encoder_blocks.0.feedforward._linear_layers.1.weight", "_modeling_layer._encoder_blocks.0.feedforward._linear_layers.1.bias", "_modeling_layer._encoder_blocks.1._conv_norm_layers.0.weight", "_modeling_layer._encoder_blocks.1._conv_norm_layers.0.bias", "_modeling_layer._encoder_blocks.1._conv_norm_layers.1.weight", "_modeling_layer._encoder_blocks.1._conv_norm_layers.1.bias", "_modeling_layer._encoder_blocks.1._conv_layers.0.1.weight", "_modeling_layer._encoder_blocks.1._conv_layers.0.1.bias", "_modeling_layer._encoder_blocks.1._conv_layers.0.2.weight", "_modeling_layer._encoder_blocks.1._conv_layers.0.2.bias", "_modeling_layer._encoder_blocks.1._conv_layers.1.1.weight", "_modeling_layer._encoder_blocks.1._conv_layers.1.1.bias", "_modeling_layer._encoder_blocks.1._conv_layers.1.2.weight", "_modeling_layer._encoder_blocks.1._conv_layers.1.2.bias", "_modeling_layer._encoder_blocks.1.attention_norm_layer.weight", "_modeling_layer._encoder_blocks.1.attention_norm_layer.bias", "_modeling_layer._encoder_blocks.1.attention_layer._combined_projection.weight", "_modeling_layer._encoder_blocks.1.attention_layer._combined_projection.bias", "_modeling_layer._encoder_blocks.1.attention_layer._output_projection.weight", "_modeling_layer._encoder_blocks.1.attention_layer._output_projection.bias", "_modeling_layer._encoder_blocks.1.feedforward_norm_layer.weight", "_modeling_layer._encoder_blocks.1.feedforward_norm_layer.bias", "_modeling_layer._encoder_blocks.1.feedforward._linear_layers.0.weight", "_modeling_layer._encoder_blocks.1.feedforward._linear_layers.0.bias", "_modeling_layer._encoder_blocks.1.feedforward._linear_layers.1.weight", "_modeling_layer._encoder_blocks.1.feedforward._linear_layers.1.bias", "_modeling_layer._encoder_blocks.2._conv_norm_layers.0.weight", "_modeling_layer._encoder_blocks.2._conv_norm_layers.0.bias", "_modeling_layer._encoder_blocks.2._conv_norm_layers.1.weight", "_modeling_layer._encoder_blocks.2._conv_norm_layers.1.bias", "_modeling_layer._encoder_blocks.2._conv_layers.0.1.weight", "_modeling_layer._encoder_blocks.2._conv_layers.0.1.bias", "_modeling_layer._encoder_blocks.2._conv_layers.0.2.weight", "_modeling_layer._encoder_blocks.2._conv_layers.0.2.bias", "_modeling_layer._encoder_blocks.2._conv_layers.1.1.weight", "_modeling_layer._encoder_blocks.2._conv_layers.1.1.bias", "_modeling_layer._encoder_blocks.2._conv_layers.1.2.weight", "_modeling_layer._encoder_blocks.2._conv_layers.1.2.bias", "_modeling_layer._encoder_blocks.2.attention_norm_layer.weight", "_modeling_layer._encoder_blocks.2.attention_norm_layer.bias", "_modeling_layer._encoder_blocks.2.attention_layer._combined_projection.weight", "_modeling_layer._encoder_blocks.2.attention_layer._combined_projection.bias", "_modeling_layer._encoder_blocks.2.attention_layer._output_projection.weight", "_modeling_layer._encoder_blocks.2.attention_layer._output_projection.bias", "_modeling_layer._encoder_blocks.2.feedforward_norm_layer.weight", "_modeling_layer._encoder_blocks.2.feedforward_norm_layer.bias", "_modeling_layer._encoder_blocks.2.feedforward._linear_layers.0.weight", "_modeling_layer._encoder_blocks.2.feedforward._linear_layers.0.bias", "_modeling_layer._encoder_blocks.2.feedforward._linear_layers.1.weight", "_modeling_layer._encoder_blocks.2.feedforward._linear_layers.1.bias", "_modeling_layer._encoder_blocks.3._conv_norm_layers.0.weight", "_modeling_layer._encoder_blocks.3._conv_norm_layers.0.bias", "_modeling_layer._encoder_blocks.3._conv_norm_layers.1.weight", "_modeling_layer._encoder_blocks.3._conv_norm_layers.1.bias", "_modeling_layer._encoder_blocks.3._conv_layers.0.1.weight", "_modeling_layer._encoder_blocks.3._conv_layers.0.1.bias", "_modeling_layer._encoder_blocks.3._conv_layers.0.2.weight", "_modeling_layer._encoder_blocks.3._conv_layers.0.2.bias", "_modeling_layer._encoder_blocks.3._conv_layers.1.1.weight", "_modeling_layer._encoder_blocks.3._conv_layers.1.1.bias", "_modeling_layer._encoder_blocks.3._conv_layers.1.2.weight", "_modeling_layer._encoder_blocks.3._conv_layers.1.2.bias", "_modeling_layer._encoder_blocks.3.attention_norm_layer.weight", "_modeling_layer._encoder_blocks.3.attention_norm_layer.bias", "_modeling_layer._encoder_blocks.3.attention_layer._combined_projection.weight", "_modeling_layer._encoder_blocks.3.attention_layer._combined_projection.bias", "_modeling_layer._encoder_blocks.3.attention_layer._output_projection.weight", "_modeling_layer._encoder_blocks.3.attention_layer._output_projection.bias", "_modeling_layer._encoder_blocks.3.feedforward_norm_layer.weight", "_modeling_layer._encoder_blocks.3.feedforward_norm_layer.bias", "_modeling_layer._encoder_blocks.3.feedforward._linear_layers.0.weight", "_modeling_layer._encoder_blocks.3.feedforward._linear_layers.0.bias", "_modeling_layer._encoder_blocks.3.feedforward._linear_layers.1.weight", "_modeling_layer._encoder_blocks.3.feedforward._linear_layers.1.bias", "_modeling_layer._encoder_blocks.4._conv_norm_layers.0.weight", "_modeling_layer._encoder_blocks.4._conv_norm_layers.0.bias", "_modeling_layer._encoder_blocks.4._conv_norm_layers.1.weight", "_modeling_layer._encoder_blocks.4._conv_norm_layers.1.bias", "_modeling_layer._encoder_blocks.4._conv_layers.0.1.weight", "_modeling_layer._encoder_blocks.4._conv_layers.0.1.bias", "_modeling_layer._encoder_blocks.4._conv_layers.0.2.weight", "_modeling_layer._encoder_blocks.4._conv_layers.0.2.bias", "_modeling_layer._encoder_blocks.4._conv_layers.1.1.weight", "_modeling_layer._encoder_blocks.4._conv_layers.1.1.bias", "_modeling_layer._encoder_blocks.4._conv_layers.1.2.weight", "_modeling_layer._encoder_blocks.4._conv_layers.1.2.bias", "_modeling_layer._encoder_blocks.4.attention_norm_layer.weight", "_modeling_layer._encoder_blocks.4.attention_norm_layer.bias", "_modeling_layer._encoder_blocks.4.attention_layer._combined_projection.weight", "_modeling_layer._encoder_blocks.4.attention_layer._combined_projection.bias", "_modeling_layer._encoder_blocks.4.attention_layer._output_projection.weight", "_modeling_layer._encoder_blocks.4.attention_layer._output_projection.bias", "_modeling_layer._encoder_blocks.4.feedforward_norm_layer.weight", "_modeling_layer._encoder_blocks.4.feedforward_norm_layer.bias", "_modeling_layer._encoder_blocks.4.feedforward._linear_layers.0.weight", "_modeling_layer._encoder_blocks.4.feedforward._linear_layers.0.bias", "_modeling_layer._encoder_blocks.4.feedforward._linear_layers.1.weight", "_modeling_layer._encoder_blocks.4.feedforward._linear_layers.1.bias", "_modeling_layer._encoder_blocks.5._conv_norm_layers.0.weight", "_modeling_layer._encoder_blocks.5._conv_norm_layers.0.bias", "_modeling_layer._encoder_blocks.5._conv_norm_layers.1.weight", "_modeling_layer._encoder_blocks.5._conv_norm_layers.1.bias", "_modeling_layer._encoder_blocks.5._conv_layers.0.1.weight", "_modeling_layer._encoder_blocks.5._conv_layers.0.1.bias", "_modeling_layer._encoder_blocks.5._conv_layers.0.2.weight", "_modeling_layer._encoder_blocks.5._conv_layers.0.2.bias", "_modeling_layer._encoder_blocks.5._conv_layers.1.1.weight", "_modeling_layer._encoder_blocks.5._conv_layers.1.1.bias", "_modeling_layer._encoder_blocks.5._conv_layers.1.2.weight", "_modeling_layer._encoder_blocks.5._conv_layers.1.2.bias", "_modeling_layer._encoder_blocks.5.attention_norm_layer.weight", "_modeling_layer._encoder_blocks.5.attention_norm_layer.bias", "_modeling_layer._encoder_blocks.5.attention_layer._combined_projection.weight", "_modeling_layer._encoder_blocks.5.attention_layer._combined_projection.bias", "_modeling_layer._encoder_blocks.5.attention_layer._output_projection.weight", "_modeling_layer._encoder_blocks.5.attention_layer._output_projection.bias", "_modeling_layer._encoder_blocks.5.feedforward_norm_layer.weight", "_modeling_layer._encoder_blocks.5.feedforward_norm_layer.bias", "_modeling_layer._encoder_blocks.5.feedforward._linear_layers.0.weight", "_modeling_layer._encoder_blocks.5.feedforward._linear_layers.0.bias", "_modeling_layer._encoder_blocks.5.feedforward._linear_layers.1.weight", "_modeling_layer._encoder_blocks.5.feedforward._linear_layers.1.bias". 
```

**Comment by matt-gardner** [comment]
I'm seeing the same error.  Looking into it.

**Comment by matt-gardner** [comment]
It was #2692 that introduced this bug.  Looking into how to fix it.

**Comment by matt-gardner** [comment]
Fix in #2773.  @Phirefly9, let me know if that works for you.

**Comment by Phirefly9** [comment]
@matt-gardner The model initialized on that commit. Thank you for the fast turn-around
